readme.md

```diff
@@ -8,7 +8,12 @@ For the following quiz, there will be a series of questions or directions follow
 What is Git, what problem does it solve?
 
 ```
-answer goes here
+Git is a distributed version control system. It simultaneously provides a solution
+to back up code as it is written and to allow teams of multiple programmers to work
+on one code base simultaneously.
+
+It preserves each stage of development automatically so programmers do not have
+to create different versions of code manually as experimental features are developed.
 
 ```
 
```

Comment:
I would quibble with "automatically" since we have to stage and commit files, but your answer is essentially correct. 👍